partition_snapshot_reader.hh

```diff
@@ -339,6 +344,8 @@ private:
     std::optional<query::clustering_range> opt_reversed_range;
 
     std::optional<position_in_partition> _last_entry;
+    // When not Reversing, it's .position() of last emitted range tombstone.
+    // When Reversing, it's .end_position() of rt which is still in the snapshot domain.
```

Comment:
This comment is really confusing

Reply:
It would be more helpful if the comment summarized the higher order meaning of last_rts rather than how it is computed. For example: "The range [-inf, last_rts], in snapshot schema domain, is the range of range tombstone information yet to be emitted by the reader. All information relevant to (last_rts, +inf] was already emitted."